robotoff/insights/question.py

```diff
@@ -164,6 +170,47 @@ def get_source_image_url(barcode: str) -> Optional[str]:
 
         return None
 
+    @staticmethod
+    def generate_selected_images(images: JSONType, barcode: str) -> JSONType:
+        selected_images = {}
+        images_data = {}
+        front = {}
+        small = {}
+        thumb = {}
+        display = {}
+
+        splitted_barcode = split_barcode(barcode)  # splitting the barcode
+
+        for k in images.items():
+            key = k[0]
+            if key.startswith("front_"):  # to add support for all languages
```

Comment:
Why only front? You can loop over ("front", "nutrition", "ingredients", "packaging")